[2023-07-16 00:07:03,276] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: probando_conexion.primera_tarea manual__2023-07-16T00:07:02.352464+00:00 [queued]>
[2023-07-16 00:07:03,291] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: probando_conexion.primera_tarea manual__2023-07-16T00:07:02.352464+00:00 [queued]>
[2023-07-16 00:07:03,291] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-07-16 00:07:03,291] {taskinstance.py:1377} INFO - Starting attempt 1 of 11
[2023-07-16 00:07:03,291] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-07-16 00:07:03,307] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): primera_tarea> on 2023-07-16 00:07:02.352464+00:00
[2023-07-16 00:07:03,314] {standard_task_runner.py:52} INFO - Started process 5450 to run task
[2023-07-16 00:07:03,317] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'probando_conexion', 'primera_tarea', 'manual__2023-07-16T00:07:02.352464+00:00', '--job-id', '848', '--raw', '--subdir', 'DAGS_FOLDER/variables_y_secrets.py', '--cfg-path', '/tmp/tmpiwc4ucx2', '--error-file', '/tmp/tmpfn8vv0f6']
[2023-07-16 00:07:03,318] {standard_task_runner.py:80} INFO - Job 848: Subtask primera_tarea
[2023-07-16 00:07:03,397] {task_command.py:371} INFO - Running <TaskInstance: probando_conexion.primera_tarea manual__2023-07-16T00:07:02.352464+00:00 [running]> on host 7a1b6d75866e
[2023-07-16 00:07:03,539] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=arielmfeldman.af@gmail.com
AIRFLOW_CTX_DAG_OWNER=Ariel Feldman
AIRFLOW_CTX_DAG_ID=probando_conexion
AIRFLOW_CTX_TASK_ID=primera_tarea
AIRFLOW_CTX_EXECUTION_DATE=2023-07-16T00:07:02.352464+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-07-16T00:07:02.352464+00:00
[2023-07-16 00:07:03,559] {logging_mixin.py:115} INFO - no se accedio a la variable
[2023-07-16 00:07:03,559] {python.py:173} INFO - Done. Returned value was: None
[2023-07-16 00:07:03,586] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=probando_conexion, task_id=primera_tarea, execution_date=20230716T000702, start_date=20230716T000703, end_date=20230716T000703
[2023-07-16 00:07:03,611] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-07-16 00:07:03,654] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
